cogboard-app/src/main/kotlin/com/cognifide/cogboard/security/LoginHandler.kt

```diff
@@ -28,34 +32,36 @@ class LoginHandler(val storage: Storage = VolumeStorageFactory.admins()) : Routi
 
         return Handler { ctx ->
             ctx.bodyAsJson?.let {
-                val username = it.getString("username", "")
-                val password = it.getString("password", "")
+                val username = it.getString(PROP_USERNAME, "")
+                val password = it.getString(PROP_PASSWORD, "")
                 val admin = getAdmin(username)
                 when {
                     admin == null -> sendUnauthorized(ctx, wrongUserMsg)
-                    admin.isNotAuthorized(password) -> sendUnauthorized(ctx, wrongPassMsg)
+                    isNotAuthorized(admin, password) -> sendUnauthorized(ctx, wrongPassMsg)
                     else -> sendJWT(ctx, username)
                 }
             }
         }
     }
 
     private fun getAdmin(name: String): Admin? {
-        val data = storage
-                .loadConfig()
-                .getJsonArray("admins")
-                ?.map { it as JsonObject }
-                ?.firstOrNull { it.getString("name") == name }
-        return if (data != null) Admin(data.getString("name"), data.getString("pass"))
-        else null
+        val admin = storage.loadConfig()
+        val username = admin.getString(PROP_USER)
+        return if (username == name) {
+            val password = admin.getString(PROP_PASSWORD)
+            Admin(username, password)
+        } else null
     }
 
-    private fun sendJWT(ctx: RoutingContext, user: String) {
-        ctx.response().end(generateJWT(user))
-    }
+    private fun isNotAuthorized(admin: Admin, password: String?) =
```

Comment:
can we revert this - it is always a better practise to use positive methods without negation:
```
private fun isAuthorized(admin: Admin, password: String?) = 
    admin.password.isNotBlank() && admin.password == password
```

Reply:
fixed